src/composition.rs

```diff
@@ -299,6 +334,7 @@ impl CompositionQueryAnalyzer {
                 )
             }
             Some(measurement) => {
+                let measurements = vec![measurement.to_owned()];
```

Comment:
Note: allowing multiple measurements is a new requirement as of this week.

Reply:
Let's specifically remove this, and not support multiple measurements until we have official scope, talked about design and behavior, and worked that through. We have a deadline we have to deliver by, and we should stick to the feature set we've committed to until we've delivered that.

Reply:
removed. Is now passed inline as a vector slice (since that is required for the recursive `logical_expr()`)